unitary/alpha/qudit_gates.py

```diff
@@ -16,7 +16,43 @@
 import cirq
 
 
+class QuditXGate(cirq.Gate):
+    """Performs a X_0a gate.
```

Comment:
Would it make sense to have a general X_{ab} gate instead (where a is not always 0, I mean)?

Reply:
Done.